obd/decoders.py

```diff
@@ -8,6 +8,7 @@
 # Copyright 2009 Secons Ltd. (www.obdtester.com)                       #
 # Copyright 2009 Peter J. Creath                                       #
 # Copyright 2016 Brendan Whitfield (brendan-w.com)                     #
+# Copyright 2019 Adaptant Solutions AG                                 #
```

Comment:
Is additional copyright needed?

Reply:
Our contribution, our copyright. What exactly do you perceive to be the issue?

Reply:
Usually a files copyright is only changed with a major change, like a full re-write. I'm not sure if this is enough of a change to justify adding your copyright

Reply:
Contributions are generally made under the same license as the project (unless different agreement). There are [GitHub policies for contributions](https://github.com/github/site-policy/blob/master/Policies/github-terms-of-service.md#6-contributions-under-repository-license).

`git blame` can always be used to view the revision history of lines within a file, and [GitHub *blame*](https://help.github.com/en/articles/tracking-changes-in-a-file) shows the latest line-by-line changes made by respective authors in a file, like here:
https://github.com/brendan-w/python-OBD/blame/8e044f5a797a7ec2ec1341d42250af0bf654ac14/obd/decoders.py